pkg/question/question.go

```diff
@@ -221,8 +226,13 @@ func AskLaunchTemplate(h *ec2helper.EC2Helper) *string {
 	data := [][]string{}
 	indexedOptions := []string{}
 
+	noUseOptionRepr, noUseOptionValue := "Do not use launch template", cli.ResponseNo
```

Comment:
What does `Repr` stand for? I see it used all over the place, so if we want to change it, that should be a separate refactor PR. But it's a bit ambiguous to me.